gobblin-runtime/src/test/java/org/apache/hadoop/fs/FileSystemTestUtils.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package org.apache.hadoop.fs;
+
+import java.io.IOException;
+import java.net.URI;
+
+import org.apache.hadoop.conf.Configuration;
+
+
+public class FileSystemTestUtils {
+
+  public static void addFileSystemForTest(URI uri, Configuration conf, FileSystem fs) throws IOException {
```

Comment:
Because the method is protected, so it needs to be in the correct package (`org.apache.hadoop.fs`)